baseplate/lib/thrift_pool.py

```diff
@@ -24,7 +25,6 @@
 from typing import Optional
 from typing import TYPE_CHECKING
 
-from gevent import queue as gevent_queue
```

Comment:
Undoing explicitly using a gevent queue, because the standard library queue should be patched.

https://github.com/reddit/baseplate.py/pull/643